Core/TextureReplacer.h

```diff
@@ -169,12 +201,13 @@ class TextureReplacer {
 	std::string gameID_;
 	std::string basePath_;
 	ReplacedTextureHash hash_;
-	std::unordered_map<std::string, std::string> aliases_;
 	typedef std::pair<int, int> WidthHeightPair;
 #ifdef __SYMBIAN32__
```

Comment:
Yeah... but you also have to define `<`, so it'll still bite us...

-[Unknown]